linux_os/guide/system/software/integrity/software-integrity/aide/aide_periodic_checking_systemd_timer/ansible/sle15.yml

```diff
@@ -0,0 +1,49 @@
+# platform = multi_platform_sle
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+- name: "{{{ rule_title }}} - Ensure AIDE is Installed"
+  ansible.builtin.package:
+    name:
+      - aide
+    state: present
+
+- name: "{{{ rule_title }}} - Define AIDE Periodic Check Service"
+  ansible.builtin.blockinfile:
+      create: yes
+      dest: /etc/systemd/system/aidecheck.service
+      block: |
+        [Unit]
+        Description=Aide Check
+        [Service]
+        Type=simple
+        ExecStart=/usr/sbin/aide --check
+        [Install]
+        WantedBy=multi-user.target
+
+- name: "{{{ rule_title }}} - Define AIDE Periodic Check Service Timer"
+  ansible.builtin.blockinfile:
```

Comment:
Would you like to ensure mode and ownership here too, like in the Bash remediation?